doc/workshops/financial-analysis.py

```diff
@@ -0,0 +1,405 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Financial Data Analysis with Spyder
+
+Created on Thu May 20 10:17:27 2021
+
+@author: Spyder Team
+"""
+
+
+# %% Import libraries
+import numpy as np
+import pandas as pd
+from pylab import mpl
```

Comment:
Honestly, I don't know. Shall I change it?

Reply:
I'd say `import matplotlib as mpl` is less confusing because it may seem like pylab is a whole different module that people don't know about. But it's up to you :)

Reply:
I agree with Melissa: `import matplotlib as mpl` is more standard than `from pylab import mpl`.

@Linus-Albertus, could you make that change?

Reply:
Ok!